src/main/java/uk/gov/hmcts/darts/audio/service/impl/MediaRequestServiceImpl.java

```diff
@@ -3,18 +3,25 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.darts.audio.entity.MediaRequest;
-import uk.gov.hmcts.darts.audio.enums.AudioRequestStatus;
-import uk.gov.hmcts.darts.audio.repository.AudioRequestRepository;
-import uk.gov.hmcts.darts.audio.service.AudioRequestService;
+import uk.gov.hmcts.darts.audio.repository.MediaRequestRepository;
+import uk.gov.hmcts.darts.audio.service.MediaRequestService;
 import uk.gov.hmcts.darts.audiorequest.model.AudioRequestDetails;
+import uk.gov.hmcts.darts.audiorequest.model.AudioRequestType;
 
 import java.time.OffsetDateTime;
 
+import static uk.gov.hmcts.darts.audio.enums.AudioRequestStatus.OPEN;
+
 @RequiredArgsConstructor
 @Service
-public class AudioRequestServiceImpl implements AudioRequestService {
+public class MediaRequestServiceImpl implements MediaRequestService {
+
+    private final MediaRequestRepository mediaRequestRepository;
 
-    private final AudioRequestRepository audioRequestRepository;
+    @Override
+    public MediaRequest getMediaRequestById(Integer id) {
+        return mediaRequestRepository.getReferenceById(id);
```

Comment:
I am assuming that the use-case for this endpoint will be to get the data of the whole MediaRequest object, instead of just its ID, so assume we will want to do a `find` here.

Reply:
I don't really understand this comment - this is getting the whole MediaRequest object?
I only added this service here because of the ticket description having
`getAudioRequest (requestId) - Fetch the Audio Request from Audio Request Table.`

Where requestId, we understand to be the PK :)
FYI @hemantasharma1129

Reply:
there is a functional difference between `getReferenceById` and `findById`.
`getReferenceById` returns a proxy version of the entity, and would be used when you only want to link it to another entity, and you don't want to get or set any of the columns on it, but just refer to it.
This might explain it better
https://www.baeldung.com/jpa-entity-manager-get-reference